frontend/index.html

```diff
@@ -75,7 +75,7 @@ <h3 class="text-muted">Guestbook</h3>
                 <button type="submit"
                         class="btn btn-success btn-lg btn-block"
                         ng-click="guestbookCtlr.submitForm()"
-                        ng-disabled="formdata.user.location === null || !formdata.user.name || !formdata.text">Submit your post!</button>
+                        ng-disabled="!formdata.user.name || !formdata.text">Submit your post!</button>
```

Comment:
This allows the user to submit the form, even when the browser's geolocation API does not yield a response.